### PR TITLE
Add statement on removal of Broyden acceleration

### PR DESCRIPTION
**Pull request by Fujikawas**
Followed up on https://github.com/precice/precice/pull/1735

### CODE REVIEW

File: pages/docs/couple-your-code/couple-your-code-porting-v2-3.md [hunk 1]
```diff
@@ -138,6 +138,8 @@ A specific solver should only be configured if you want to force preCICE to use
   - Removed callback functions `vertexCallback` and `postAction` from `PythonAction` interface.
   - Removed timewindowsize from the `performAction` signature of `PythonAction`. The new signature is `performAction(time, data)`
 
+- We removed the Broyden acceleration. Use IMVJ for Broyden-like acceleration instead.
```
Comment (MakisH) [inline pages/docs/couple-your-code/couple-your-code-porting-v2-3.md]: What does "Broyden-like" mean? Can we cite some reference here?

Comment (uekerman) [inline pages/docs/couple-your-code/couple-your-code-porting-v2-3.md]:
```suggestion
- We removed the plain `Broyden` acceleration. You could use `IQN-IMVJ` instead, which is a [multi-vector Broyden variant](http://hdl.handle.net/2117/191193).
```

Comment (uekerman) [reply pages/docs/couple-your-code/couple-your-code-porting-v2-3.md]: I extended the comment. Understandable like this?

Comment (MakisH) [reply pages/docs/couple-your-code/couple-your-code-porting-v2-3.md]: yes, much better, thanks!

**Review by MakisH — APPROVED** [review]

**Pull request merged**